.github/workflows/release.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       # Checks out the repository
       - uses: actions/checkout@v3
         with:
-          ref: 'master'
+          ref: "master"
 
       # Upgrade pip
       - name: Upgrade pip
```

Comment:
Why the changes in this file?

Reply:
In order to resolve the warnings you have in the actions.
https://github.com/aws-samples/amazon-textract-textractor/actions/runs/3915318666